tests/countries/test_pitcairn_islands.py

```diff
@@ -0,0 +1,75 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+import unittest
+
+from holidays.countries.pitcairn_islands import PitcairnIslands, PN, PCN
+from tests.common import CommonCountryTests
+
+
+class TestPitcairnIslands(CommonCountryTests, unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(PitcairnIslands, years=range(2000, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(PitcairnIslands, PN, PCN)
+
+    def test_2024(self):
+        self.assertHolidays(
+            PitcairnIslands(years=2024),
+            ("2024-01-01", "New Year's Day"),
+            ("2024-01-23", "Bounty Day"),
+            ("2024-03-29", "Good Friday"),
+            ("2024-04-01", "Easter Monday"),
+            ("2024-06-08", "King's Birthday"),  # 2nd Saturday in June
```

Comment:
Consider looking at #2716 for standard test case implementation, and for category tests, [see Libya's implementation](https://github.com/vacanza/holidays/blob/5b9e6e60bd4c28772f7d6e7b6184442f76af33c7/tests/countries/test_libya.py#L25)